src/routes/transactions/transactions.controller.ts

```diff
@@ -50,4 +52,36 @@ export class TransactionsController {
       paginationData,
     );
   }
+
+  @ApiOkResponse({ type: IncomingTransferPage })
+  @Get('chains/:chainId/safes/:safeAddress/incoming-transfers')
```

Comment:
We could wrap all the route params into a DTO here for the sake of brevity/clarity. Since this could also be done for other routes, probably it's better to tackle it in a separate issue, wdyt?

Reply:
No strong opinion on this one. I think for now this works as it also reduces the coupling to the `common` package (which would contain those DTOs)